components/ListView/ComponentDetailsView.js

```diff
@@ -339,14 +328,12 @@ class ComponentDetailsView extends React.Component {
               </dl>
             </Tab>
             {this.state.componentData.components &&
-              <Tab tabTitle="Components" active={this.state.activeTab === 'Components'}>
+              <Tab eventKey="components" title="Components">
                 <p><FormattedMessage defaultMessage="Components" /></p>
               </Tab>}
             <Tab
-              tabTitle={formatMessage(messages.dependencies,
-                {count: `<span class="badge">${this.state.dependencies.length}</span>`}
-              )}
-              active={this.state.activeTab === 'Dependencies'}
+              eventKey="dependencies"
+              title={<LabelWithBadge title={formatMessage(messages.dependencies)} bagde={this.state.dependencies.length} />}
```

Comment:
Oh, maybe this typo is causing the dependencies count to not display: `bagde`?

Reply:
Oops, indeed. Thank you!